.github/workflows/source-web-tool-scripts.yaml

```diff
@@ -90,9 +90,19 @@ jobs:
           install.packages(setdiff(deps, exclude))
         shell: Rscript {0}
 
-      - name: Source web-tool scripts
+      - name: Source web-tool scripts with latest packages
         run: |
           setwd("PACTA_analysis")
           source("R/source_web_tool_scripts.R")
           source_web_tool_scripts()
         shell: Rscript {0}
+      - name: Source web-tool scripts with versioned packages
+        run: |
+          # See #435
+          devtools::install_version("readr", "1.3.1")
+          stopifnot(utils::packageVersion("readr") < "1.4.0")
+
+          setwd("PACTA_analysis")
+          source("R/source_web_tool_scripts.R")
+          source_web_tool_scripts()
+        shell: Rscript {0}
```

Comment:
@AlexAxthelm, here is the bit where I extend the existing workflow to test the web tool not only with the latest packages but also with a version of readr that relates to this PR. If we need to test with other versioned-packages, we add the relevant calls to `remotes::install_version()` here.

Where you thinking of some other strategy? Maybe using a docker image on gh with the specific package-versions already installed?

Reply:
be aware that this repo is still private and this will probably double the GH Action time we're using

Reply:
Yeah, I'm inclined to keep this as a "latest-only" check, and we can discuss a strategy for checking older versions later? I'll flag an issue for it on ADO.

Reply:
> be aware that this repo is still private

Do you mean this repo = PACTA_analysis? It's not private, is it?

Reply:
Ok, I removed the code that tests the web tool with readr < 1.4.0

Reply:
> Do you mean this repo = PACTA_analysis? It's not private, is it?

you're right, sorry, my mistake